lib/rules/execute-rule-includes-list-all.js

```diff
@@ -0,0 +1,21 @@
+const includes = require('lodash.includes');
+const prepForComparison = require('./prep-for-comparison');
+
+function executeRuleIncludesListAll(input, rule, values) {
+  const firstMatch = values.find((value) => {
+    const property = value[rule.property];
```

Comment:
should this name be `propertyValues`? Since we use that same variable name here in`executeRuleList`: https://github.com/godaddy/node-flipr/pull/15/files#diff-45a8bd2a3f45fde238f27bb6e1250014R5`